dotCMS/src/main/java/com/dotcms/util/user/LiferayUserTransformer.java

```diff
@@ -0,0 +1,164 @@
+            }
+            if (null != map.get("languageid")) {
+                user.setLanguageId((String) map.get("languageid"));
+            }
+            if (null != map.get("timezoneid")) {
+                user.setTimeZoneId((String) map.get("timezoneid"));
+            }
+            if (null != map.get("skinid")) {
+                user.setSkinId((String) map.get("skinid"));
+            }
+            if (null != map.get("dottedskins")) {
+                user.setDottedSkins(
+                        DbConnectionFactory.isDBTrue(map.get("dottedskins").toString()));
+            }
+            if (null != map.get("roundedskins")) {
+                user.setRoundedSkins(
+                        DbConnectionFactory.isDBTrue(map.get("roundedskins").toString()));
+            }
+            if (null != map.get("greeting")) {
+                user.setGreeting((String) map.get("greeting"));
+            }
+            if (null != map.get("resolution")) {
+                user.setResolution((String) map.get("resolution"));
+            }
+            if (null != map.get("refreshrate")) {
+                user.setRefreshRate((String) map.get("refreshrate"));
+            }
+            if (null != map.get("layoutids")) {
+                user.setLayoutIds((String) map.get("layoutids"));
+            }
+            if (null != map.get("comments")) {
+                user.setComments((String) map.get("comments"));
+            }
+            if (null != map.get("logindate")) {
+                user.setLoginDate((java.util.Date) map.get("logindate"));
+            }
+            if (null != map.get("loginip")) {
+                user.setLoginIP((String) map.get("loginip"));
+            }
+            if (null != map.get("lastlogindate")) {
+                user.setLastLoginDate((java.util.Date) map.get("lastlogindate"));
+            }
+            if (null != map.get("lastloginip")) {
+                user.setLastLoginIP((String) map.get("lastloginip"));
+            }
+            if (null != map.get("failedloginattempts")) {
+                user.setFailedLoginAttempts((Integer) map.get("failedloginattempts"));
+            }
+            if (null != map.get("agreedtotermsofuse")) {
+                user.setAgreedToTermsOfUse(
+                        DbConnectionFactory.isDBTrue(map.get("agreedtotermsofuse").toString()));
+            }
+            if (null != map.get("active_")) {
+                user.setActive(DbConnectionFactory.isDBTrue(map.get("active_").toString()));
+            }
+            if (null != map.get("delete_in_progress")) {
+                user.setDeleteInProgress(DbConnectionFactory.isDBTrue(map.get("delete_in_progress")
+                        .toString()));
+            }
+            if (null != map.get("delete_date")) {
+                user.setDeleteDate((java.util.Date) map.get("delete_date"));
+            }
+
+            UserPool.put(user.getPrimaryKey(), user);
```

Comment:
I am not pretty sure this should be done here, probably is not part of his responsibility.

Reply:
done